test/fuzz/RedeemTest.t.sol

```diff
@@ -439,7 +442,16 @@ contract RedeemTest is Fixture, FunctionUtils {
                 );
                 assertApproxEqAbs(totalStable, mintedStables - amountBurnt, 3 wei);
             }
-            mintedStables = transmuter.getTotalIssued();
+
+            // Compute mintedStables while rounding up
+            uint128 normalizedStables = uint128(
```

Comment:
Was it breaking the test? 

Better to use the global system count than recomputing

Reply:
It was breaking the test as getTotalIssued only returns the rounding down result but we need the rounding up result